### PR TITLE
fix(aws/loadbalancer): ensure timeout < interval

### PR DESCRIPTION
**Pull request by icfantv**
* add an error check and message around ensuring the load balancer
health timeout is less than the health interval.
* updated help bubble text to include field value requirements for timeout and interval.
* fix broken help bubbles for classic ELB advanced settings.

### CODE REVIEW

File: app/scripts/modules/amazon/src/loadBalancer/configure/classic/createClassicLoadBalancer.controller.ts [hunk 1]
```diff
@@ -149,11 +150,16 @@ export class CreateClassicLoadBalancerCtrl extends CreateAmazonLoadBalancerCtrl
...
+
+  public validateTimeout(value: number): boolean {
+    return value < this.loadBalancerCommand.healthInterval;
+  }
 }
 
 export const AWS_CREATE_CLASSIC_LOAD_BALANCER_CTRL = 'spinnaker.amazon.loadBalancer.classic.create.controller';
 module(AWS_CREATE_CLASSIC_LOAD_BALANCER_CTRL, [
   require('@uirouter/angularjs').default,
+  'ui.validate',
```
Comment (jrsquared) [inline app/scripts/modules/amazon/src/loadBalancer/configure/classic/createClassicLoadBalancer.controller.ts]: Is there a reason we couldn't follow the existing pattern of just having the require here?

File: app/scripts/modules/amazon/src/loadBalancer/configure/classic/advancedSettings.html [hunk 2]
```diff
@@ -3,16 +3,24 @@
...
+        <b>Timeout</b><help-field key="loadBalancer.advancedSettings.healthTimeout"/>
       </div>
       <div class="col-md-4">
-        <input class="form-control input-sm" type="number" min="0" ng-model="ctrl.loadBalancerCommand.healthTimeout"/>
+        <input class="form-control input-sm"
+               type="number"
+               min="0"
+               name="healthTimeout"
+               ng-model="ctrl.loadBalancerCommand.healthTimeout"
+               ui-validate="{ timeout: 'ctrl.validateTimeout($value)' }"/>
```
Comment (anotherchrisberry) [inline app/scripts/modules/amazon/src/loadBalancer/configure/classic/advancedSettings.html]: any reason not to just use `ng-min="ctrl.loadBalancerCommand.healthInterval + 1"`? Would save pulling in another library.

Comment (icfantv) [inline app/scripts/modules/amazon/src/loadBalancer/configure/classic/advancedSettings.html]: Oh.....clever.  I was trying to think of a way to not have to pull in something.  Lemme test that out.

File: app/scripts/modules/amazon/src/loadBalancer/configure/classic/createClassicLoadBalancer.controller.ts (hunk 1 above)
Comment (icfantv) [inline app/scripts/modules/amazon/src/loadBalancer/configure/classic/createClassicLoadBalancer.controller.ts]: Yea, it doesn't export the module so require doesn't work.  this lib is...old...  But, good thing is if @anotherchrisberry's suggestion works, we don't need it.

**Comment by icfantv** [comment]
Much better.

**Pull request merged**